app/src/main/java/com/ajdi/yassin/popularmovies/ui/movieslist/MoviesActivity.java

```diff
@@ -1,32 +1,29 @@
 package com.ajdi.yassin.popularmovies.ui.movieslist;
-
+import android.content.Intent;
 import android.os.Bundle;
 import android.view.MenuItem;
-
+import com.ajdi.yassin.popularmovies.LoadingActivity;
 import com.ajdi.yassin.popularmovies.R;
 import com.ajdi.yassin.popularmovies.ui.movieslist.discover.DiscoverMoviesFragment;
 import com.ajdi.yassin.popularmovies.ui.movieslist.favorites.FavoritesFragment;
 import com.ajdi.yassin.popularmovies.utils.ActivityUtils;
 import com.google.android.material.bottomnavigation.BottomNavigationView;
-
 import androidx.annotation.NonNull;
 import androidx.appcompat.app.AppCompatActivity;
 import androidx.appcompat.widget.Toolbar;
-
 public class MoviesActivity extends AppCompatActivity {
-
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
-
         if (savedInstanceState == null) {
+            Intent intent = new Intent(this, LoadingActivity.class);
```

Comment:
The reason why this function is added to this is because you can skip by pressing the Cancel button if the loading screen feels like a long time.